README.md

```diff
@@ -165,7 +180,9 @@ net.export_hdf5('network.net')
 
 ## __`lava.lib.dl.bootstrap`__
 
-`lava.lib.dl.bootstrap` enables rapid training of rate based SNNs by translating them to equivalent dynamic ANN representation which leads to SNN performance close to the equivalent ANN and low latency inference. It also supports _hybrid training_ with ANN-SNN mixed network to minimize the ANN to SNN performance gap. This method is independent of the SNN model being used.
+In general ANN-SNN conversion methods for rate based SNN result in high latency of the network. This is because the rate interpretation of a spiking neuron using ReLU acitvation unit breaks down for short inference times. 
```

Comment:
high latency during inference

breaks down and then requires many time steps per sample to achieve adequate inference results.